docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,87 @@
+---
+title: MUI X expands its product line with advanced date and time picker components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+After more than 18 months in our experimental lab, the date and time picker components have found a new home as part of MUI X.
+
+This means we'll be dedicating even more time and effort to these complex components, to better meet the needs of both you and your users.
+
+## What are date and time pickers?
+
+Date and time pickers are interface controls that enable the user to select a date (or time) from a menu.
+
+Common design patterns include text inputs, dropdown lists, calendars, timelines, and scrolling pickers on mobile devices.
+
+The user may need to select an individual date/time, or a range.
+
+## What is MUI X?
+
+MUI X is a collection of advanced components built for complex use cases.
+
+As opposed to our Core library, which leans on the open-source community for support, MUI X components require several full-time developers dedicated to engineering and ongoing maintenance.
+
+MUI X components are available under two licenses:
+
+- MUI X—MIT license; free to use
```

Comment:
The hyphenation in these two bullets looks odd. Perhaps it looks better when the markdown is rendered? (There's no deploy preview so couldn't check.)

```suggestion
- MUI X is MIT licensed, so free to use.
```